tests/e2e/e2e_setup_test.go

```diff
@@ -583,6 +602,165 @@ func noRestart(config *docker.HostConfig) {
 	}
 }
 
+// hermes0 is for ibc and packet-forward-middleware test
```

Comment:
Could you add to the comment a brief explanation of why multiple Hermes instances are needed?

Reply:
we might only need one hermes which is this one in this PR. 
- before, we have one hermes running and doing ibc transfer and PFM test.
- now, i need to test hermes with two configs, one hermes' config is configured to pay fees, one not. So that I can do transactions async (with hermes not running), and i can choose which config i use to "clear" the submitted transactions. 

i will add a comment in the code, and may open an issue to refactor two instances hermes into one, to do all test without hermes running.

Reply:
That sounds reasonable, it seems out of the scope of the pr to refactor that